confluent_kafka/__init__.py

```diff
@@ -1,2 +1,2 @@
-__all__ = ['cimpl','kafkatest']
+__all__ = ['cimpl', 'avro', 'kafkatest']
```

Comment:
Should this be conditional based on install type? 
What happens if people dont install the avro submodule?